server/chainquery/models/AddressModel.js

```diff
@@ -17,15 +22,15 @@ export default (sequelize, {
       set() { },
     },
     first_seen: {
-      type: INTEGER,
+      type: DATE(6),
```

Comment:
Not sure what `Date(6)` is but the db stores a `DATETIME`

Reply:
`DATE(6)` is the equivalent